src/NavBar/NavBar.style.ts

```diff
@@ -84,10 +91,29 @@ export const NavBarHeader = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: flex-start;
+  &[data-useBreadcrumb='true'] {
+    flex-direction: row-reverse;
+    & > :last-child {
+      margin-right: 12px;
+    }
+  }
 
   & img {
     height: ${DEFAULT_SIZE}px;
     width: ${DEFAULT_SIZE}px;
     object-fit: contain;
   }
 `
+
+export const RoundIconButton = styled(BaseRoundIcon)`
+  position: absolute;
+  top: 24px;
+  left: 36px;
+  box-shadow: ${theme.shadow('regular')};
+  background-color: white;
```

Comment:
Please use theme (in case someday we have a dark mode 😈
```suggestion
  background-color: ${theme.color('background')};
```

Reply:
I can't use `${theme.color('background')}` as the background set by default is purple/primary?

Reply:
Ah ! Then you can add the param `useRoot` to use the root backgroundColor @Carolinevp